pkg/clustermesh/types/types.go

```diff
@@ -29,7 +44,8 @@ func ValidateClusterID(clusterID uint32) error {
 }
 
 type CiliumClusterConfig struct {
-	ID uint32 `json:"id,omitempty"`
+	ID                   uint32 `json:"id,omitempty"`
+	MaxConnectedClusters uint32 `json:"maxConnectedClusters,omitempty"`
```

Comment:
I wonder if this would fit more into the `Capabilities` section below.

Reply:
I think so too, I moved it there